### PR TITLE
Reworked README to be more concise

### PR DESCRIPTION
**Pull request by yanivlavi**

### CODE REVIEW

**Comment by yanivlavi** [comment]
@Akrog I rebased my changes, please review.

File: README.md [hunk 1]
```diff
@@ -30,6 +36,10 @@ Use cases:
   - VMs managed on oVirt, OpenStack and VMWare.
   - Cloud providers.
 - Take periodical snapshots of provisioned volumes.
+- Clone provisioned volumes.
```
Comment (Akrog) [inline README.md]: Should we say something like:

- Use a golden image for volume provisioning.

**Comment by Akrog** [comment]
This concise version looks better as the README.

File: README.md (hunk 1 above)
Comment (yanivlavi) [reply README.md]: I merged the two suggestions. Please review.

**Pull request merged**